.github/workflows/main.yml

```
@@ -208,11 +210,93 @@
           python -m pytest --show-capture=stderr -v -x --splits 10 --group ${{ matrix.test_group }} --splitting-algorithm=least_duration tests/tests.py
 
+  ################################################################################################
+  # Testing: Run tests using Pixi
+  ################################################################################################
+  testing-with-pixi:
+    if: github.event.pull_request.merged != true || github.ref != 'refs/heads/main'
+    strategy:
+      matrix:
+        test_group: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        # see pyprojec.toml: [tool.pixi.feature.test] for available test types
+        test_type: [test-simple, test-complex] 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)